source/guides/account/how-to-use-ledger-hardware-wallet.rst

```diff
@@ -0,0 +1,305 @@
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-7.png
+
+
+**IMPORTANT**: When initializing a transaction in your Symbol Wallet, always verify if the data (e.g. Symbol address, XYM amount) matches with the corresponding data displayed on your Ledger device. If the two data match, you can proceed with the transaction; otherwise, you should NOT continue.
+
+******************
+How to receive XYM
+******************
+
+1. Click **Accounts** in the left side panel.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-8.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-8.png
+
+2. Click the **Show on Ledger** button to verify your Symbol address. If the address in your Symbol Wallet matches with the address in your Ledger device, you can rest assured that it is true and genuine.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-9.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-9.png
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-10.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-10.png
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-11.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-11.png
+
+3. Once verified, you can now share your Symbol address with the sender.
+
+***************
+How to send XYM
+***************
+
+1. Click **Transfer**.
+
+2. Fill out the required fields.
+
+- **To** - the receiver’s XYM address.
+- **Amount of XYM** - the amount you are sending to the receiver.
+
+**IMPORTANT**: If you are sending XYM to an exchange, you might need to add an unencrypted message. Please read carefully the instructions provided by the exchange.
+
+3. Click the **Send** button. Note that there will be a fee for every transaction made.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-12.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-12.png
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-13.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-13.png
+
+4. Review each transaction detail in your Ledger device and press the right button to continue.
+
+* Recipient’s Symbol address
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-14.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-14.png
+
+* Transaction Type
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-15.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-15.png
+
+* Amount
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-16.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-16.png
+
+* Message
```

Comment:
should there be message type and message text?

Reply:
ok, splitted into two points.